### PR TITLE
PicView.PicView version 1.5.6

### PR DESCRIPTION
**Pull request by Ruben2776**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34435)

### CODE REVIEW

File: manifests/p/PicView/PicView/1.5.6/PicView.PicView.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: PicView.PicView
+PackageVersion: 1.5.6
+PackageLocale: en-US
+Publisher: PicView
+PackageName: PicView
+License: GPLv3
+ShortDescription: PicView Setup
```
Comment (ItzLevvie) [inline manifests/p/PicView/PicView/1.5.6/PicView.PicView.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Fast Picture Viewer with compact UI that can be hidden. Features image effects, galleries, EXIF and more.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/34435#pullrequestreview-801148440

**Review by ItzLevvie — APPROVED** [review]

**Comment by KaranKad** [comment]
@Ruben2776
Turns out PicView was already submitted by someone at [manifests/r/Ruben2776/PicView](https://github.com/microsoft/winget-pkgs/tree/master/manifests/r/Ruben2776/PicView), would you like to have `Ruben2776.PicView` removed or to be continued?

**Comment by Ruben2776** [comment]
@KaranKad 
Thanks for bringing that to my attention. In that case can this one `PicView.PicView` be deleted and then I can make a new Push Request to update `Ruben2776.PicView` to version 1.5.6?

**Comment by KaranKad** [comment]
Sure i'll revert this pr. Meanwhile you can update `Ruben2776.PicView` to version 1.5.6

**Pull request merged**